### ISSUE TITLE
PWA is not working 

### ISSUE THREAD

**Issue by sushilshinde** [topcoder-platform/community-app#1569]
@ThomasKranitsas PWA isn't working after I merged the PR. Can you check with @liuliquan ?

1. Offline isn't working
2. Every time I refresh, it installs new SW.

**Comment by liuliquan** [comment]
How is it deployed? In chrome console log, did you see something like this http://take.ms/cViR9 in chrome console log?

It works when visit from http://localhost:3000.

If it is visited from a domain (like local.topcoder.com), it has to be HTTPS secured.

**Comment by sushilshinde** [comment]
@liuliquan can you check with this deployment

https://beta-community-app.topcoder.com/challenges

branch is https://github.com/topcoder-platform/community-app/tree/mm-feature-experiment

**Comment by liuliquan** [comment]
I see it, fails because "Uncaught TypeError: i.registerQuotaErrorCallback is not a function"

Related to https://github.com/GoogleChrome/workbox/issues/1642

Maybe upgrade workbox-webpack-plugin to latest 3.6.1 will fix it

**Comment by liuliquan** [comment]
Ah, the package-lock.json is not updated to lock the version, that's why it gets installed with the buggy v3.5.0 ...

**Comment by sushilshinde** [comment]
Now working on your local?

Just upgraded workbox?

**Comment by liuliquan** [comment]
Yes, upgraded workbox works. It is a bug of v3.5 workbox. Either 3.4.1 (the version I used in dev) or latest 3.6.1 will work.

**Comment by sushilshinde** [comment]
@liuliquan @ThomasKranitsas 

Here is the PWA lighthouse audit, not very good it seems. What are we missing here?

<img width="1001" alt="screen shot 2018-09-24 at 5 33 12 pm" src="https://user-images.githubusercontent.com/683231/45951141-029b0380-c020-11e8-9f0d-122615de5dcd.png">

**Comment by liuliquan** [comment]
1: For page load, uncheck the "Clear Storage" button when audit. The cached storage is the key to make it faster.

This may also depends on network speed.

2, 3, 4: The manifest was added to topcoder-react-utils in 0.7.6 version. But looks it is not there in the deployed 0.7.8 version.

5: Not sure about this

**Comment by liuliquan** [comment]
I think you can lock topcoder-react-utils to 0.7.7 version to test this. The PWA manifest staff was not included in 0.7.8

**Comment by sushilshinde** [comment]
@liuliquan @ThomasKranitsas sent you guys one email, let's discuss there.

**Comment by ThomasKranitsas** [comment]
I didn't get any email
